client/src/app/ui/modules/import-list/components/import-list/import-list.component.html

```diff
@@ -194,8 +194,8 @@ <h2>{{ 'Preview' | translate }}</h2>
     <mat-icon class="newBadge" color="accent" inline *ngIf="model.willBeCreated">add</mat-icon>
 </ng-template>
 
-<ng-template #entryTemplate let-entry="entry" let-def="def" let-row="row">
-    <div *ngIf="entry[def]" class="flex-vertical-center">
+<ng-template #entryTemplate let-entry="entry" let-def="def" let-row="row" let-type="type">
+    <div *ngIf="entry[def] && type !== 'boolean'" class="flex-vertical-center">
```

Comment:
```suggestion
    <div *ngIf="entry[def] || type === 'boolean'" class="flex-vertical-center">
```
and then move the separate div below up into its own `ng-container` to deduplicate the warning icon